db/migrate/20220206065135_remove_group_name_displayed_from_assignment_properties.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveGroupNameDisplayedFromAssignmentProperties < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :assignment_properties, :group_name_displayed, :boolean
```

Comment:
Please add `default: false` and `null: false` as arguments here, so that if the migration is rolled back, the column has the same properties as before.

Reply:
Done!